### ISSUE TITLE
Newest Ubuntu apt upgrade overwrites config file in /etc/authelia

### ISSUE THREAD

**Issue by rhclayto** [authelia/authelia#2817]
### Description

I upgraded with apt on Ubuntu 20.04. After upgrade, the config file had been completely overwritten. No dialogue during the apt upgrade warned about the behavior or asked for user interaction.

### Expected Behaviour

Notify the user of impending overwrite, & ask what to do during apt upgrade.

### Reproduction Steps
`sudo apt update && sudo apt upgrade`

### Additional Information

I would have been in trouble if I didn't have backups.

**Comment by james-d-elliott** [comment]
Can you provide the output of `cat /var/lib/dpkg/info/authelia.conffiles`?

**Comment by james-d-elliott** [comment]
Looks like it doesn't get created on my end, pretty sure this is an issue with the packaging tool we use and we'll have to figure out how to fix it. Apologies if your output is the same!

**Comment by rhclayto** [comment]
`cat: /var/lib/dpkg/info/authelia.conffiles: No such file or directory`

`ls /var/lib/dpkg/info|grep authelia`:
```
authelia.list
authelia.md5sums
```

`less /var/lib/dpkg/info/authelia.list`:
```
/usr
/usr/lib
/usr/lib/systemd
/usr/lib/systemd/system
/usr/lib/systemd/system/authelia.service
/usr/bin
/usr/bin/authelia
/etc
/etc/authelia
/etc/authelia/configuration.yml
```

**Comment by james-d-elliott** [comment]
We're working on a fix with the packaging tool.

**Comment by jamesmacwhite** [comment]
This just bit me as well. Wondering if it's worth putting a note on here until it is fixed? https://apt.authelia.com/stable/debian/packages/authelia/

I appreciate the issue is known and been flagged here, but anyone using the repository has no warning of this and if using unattended-upgrades, you'd never know until it's too late.

**Comment by rhclayto** [comment]
Just updated today & this is still an issue.

**Comment by james-d-elliott** [comment]
We've managed to have this fixed upstream and it should be fixed here in the next release (or the following depending on how dpkg handles an upgrade which includes conffiles definitions).

**Comment by james-d-elliott** [comment]
Linked PR fixes this, has been tested. The deb's will be fixed in 4.34.4 which will be released once I can get the iOS webauthn issue resolved.

**Comment by jamesmacwhite** [comment]
Confirmed this is now fixed.

> Setting up authelia (4.34.5-1) ...

Configuration file '/etc/authelia/configuration.yml'
 ==> File on system created by you or by a script.
 ==> File also in package provided by package maintainer.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** configuration.yml (Y/I/N/O/D/Z) [default=N] ?
